Makefile

```diff
@@ -75,20 +75,20 @@ reqs_dev: requirements/requirements_dev.txt
 $(env_tag): requirements/requirements.txt
 	@echo "==Installing requirements.txt=="
 	pip-sync --quiet requirements/requirements.txt
-	rm -f $(env_dev_tag)
-	rm -f $(install_tag)
 	touch $(env_tag)
 
 $(env_dev_tag): requirements/requirements_dev.txt
 	@echo "==Installing requirements_dev.txt=="
 	pip-sync --quiet requirements/requirements_dev.txt
-	rm -f $(env_tag)
-	rm -f $(install_tag)
 	touch $(env_dev_tag)
 
 setup: $(env_tag)
+	@echo "Setup"
+	rm -f $(env_dev_tag)
 
 setup_dev: $(env_dev_tag)
+	@echo "Setup Dev"
+	rm -f $(env_tag)
```

Comment:
Are you sure about removing these removals? pip-sync actually removes everything and installs what specified in the given requirements file: this way, by running `setup` or `setup_dev` in a context where we already run install in the past we would end up in keeping the `install_tag` while not actually having the package installed.

Reply:
Yes, you are right. pip-sync would remove the package too. I can undo the deletion of the install_tag removal